docs/spec/v1/gitrepositories.md

```diff
@@ -106,6 +106,8 @@ Git repository.
 is not supported for SSH addresses (e.g. `user@example.com:repository.git`).
 Instead, the valid URL format is `ssh://user@example.com:22/repository.git`.
 
+If your provider gives you a ssh URL containing an extra `:` not related to a port (gitlab.com does that) then simply replace it with a `/` like this : `ssh://git@gitlab.com:root_group/a_group/your_repo.git` => `ssh://git@gitlab.com/root_group/a_group/your_repo.git`
+
```

Comment:
This is captured in the Note above about SCP, maybe replace the example in the note with the GitLab one.

Reply:
Good point, it's done. Let me know your thoughts 👍